Cargo.toml

```diff
@@ -7,11 +7,15 @@ edition = "2021"
 
 [dependencies]
 ark-serialize = { version = "0.3.0", features = ["derive"] }
-async-compatibility-layer = { git = "https://github.com/EspressoSystems/async-compatibility-layer" }
+async-compatibility-layer = { git = "https://github.com/EspressoSystems/async-compatibility-layer", tag = "1.3.0", features = [
+    "logging-utils",
+    "async-std-executor",
+    "channel-async-std",
+] }
 async-std = { version = "1.12", features = ["attributes"] }
 clap = "4.3.10"
 commit = { git = "https://github.com/EspressoSystems/commit", tag = "0.2.2" }
-contract-bindings = { path = "../contract-bindings" }
+contract-bindings = { git = "https://github.com/EspressoSystems/espresso-sequencer.git", tag = "20231003" }
```

Comment:
I think we can use the default branch here. If we use a `tag` dependabot and `cargo update` don't work and we need to manually update. We should only do that if really necessary.